templates/taxi/driver_form.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+<h1>Driver List</h1>
+<a href="{% url "taxi:driver-create" %}" class="btn btn-primary">Create Driver</a>
+<table>
+    <tr>
+        <th>Username</th>
+        <th>First name</th>
+        <th>Last name</th>
+        <th>License number</th>
+    </tr>
+    {% for driver in drivers %}
+    <tr>
+        <td>{{ driver.username }}</td>
+        <td>{{ driver.first_name }}</td>
+        <td>{{ driver.last_name }}</td>
+        <td>{{ driver.license_number }}</td>
+        <td>
+            <a href="{% url "taxi:driver-delete" driver.pk %}" class="btn btn-danger">Delete</a>
+        </td>
+    </tr>
+    {% endfor %}
+</table>
+{% endblock %}
```

Comment:
The template is misnamed as `driver_form.html`. It should be renamed to `driver_list.html` to accurately reflect its content, which is a list of drivers rather than a form.